source/common/quic/envoy_quic_client_stream.cc

```diff
@@ -407,5 +414,11 @@ void EnvoyQuicClientStream::onStreamError(absl::optional<bool> should_close_conn
 
 bool EnvoyQuicClientStream::hasPendingData() { return BufferedDataBytes() > 0; }
 
+void EnvoyQuicClientStream::enableHttpDatagramSupport() {
```

Comment:
just to check this is only currently used in unit tests right? should there be a TODO to hook it up elsewhere?

Reply:
Right, good point. This should be called when the Capsule-Protocol header field is detected with a true value. I will add a TODO for this.

Reply:
Done.